uwblueprint/rowan-house
# [RHS-138]: Add Lesson Hover Options

## Implementation Description
The lessons displayed in the edit module page should display the three icons when hovered on. These icons are: Drag Handle, Delete and Edit  Icons.  To implement this, we created a component called LessonOptions that we pass certain props to determine which lesson is being hovered on, and also to render focused and unfocused lessons differently. 

## Screenshots
![image](https://user-images.githubusercontent.com/63923219/171532246-72d69602-68b3-412e-9ef0-000b6c010f02.png)
![image](https://user-images.githubusercontent.com/63923219/171532313-17492588-46ce-42f2-aa5e-829d321bc645.png)


## What should reviewers focus on?
Reviewers should focus on checking if the code written is of high quality and ensure the functionality described above is achieved. 

## Testing Procedure
- What test(s) should we run to make sure your code works? Hover on a lesson and check if these buttons appear as shown in the design specifications. 
   - What ways could the input be wrong? N/A
   - Is it secure? Could nefarious users access/break it? N/A

## Things to Note
- Additional dependencies/libraries you've added? N/A
- Things you'd want to know when coming back to the code after a few months N/A

## Checklist
